server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -26,17 +26,13 @@
 
 public class CassandraCacheConfiguration {
 
-    private final Duration timeOut;
-    private final int byteThresholdSize;
-    private final Duration ttl;
-
     public static class Builder {
-        private final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(50);
-        private final Duration DEFAULT_TTL = Duration.ofSeconds(50);
-        private final int DEFAULT_BYTE_THRESHOLD_SIZE = 8 * 1000;
+        private static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(50);
+        private static final Duration DEFAULT_TTL = Duration.ofSeconds(50);
+        private static final int DEFAULT_BYTE_THRESHOLD_SIZE = 8 * 1000;
```

Comment:
Not really important but why not 8 * 1024 ?

Reply:
A quick study on my inbox mail headers showed most  of them are below 8K hence the value should be a good default velue regarding the objective of "not reading headers in object storage"

Reply:
On a production plateform at Linagora on a 100 sample 3% of headers exceed 8KO, 6% exceed 4KO

Reply:
You didn't get @remk point here: why not 1024 instead of 1000?